content/platform/deprecate/index.textile

```diff
@@ -0,0 +1,37 @@
+---
+title: Deprecation policy
+meta_description: "A policy detailing how Ably deprecates SDKs and APIs."
+meta_keywords: "Ably, deprecation policy, deprecation, sunset"
+---
+
+Ably's SDKs are regularly updated to add support for new features, bugfixes, and to accommodate version or API changes in dependencies. Whilst Ably ensures changes are backwards-compatible wherever possible, there are some instances where the previous API structure is no longer suitable.
+
+In all cases, updates to SDKs conform to "semantic versioning":https://semver.org/ guidelines. It is recommended that you always update to the latest version of an SDK when it is available.
+
+h2(#applicability). Applicability
+
+This policy applies to the following:
+
+Ably SDKs and associated APIs, including;
+
+* HTTP APIs
+* Ably's realtime protocol
+
+h2(#timelines). Deprecation timelines
+
+Ably will support all releases for at least 12 months from the date of release of a version that supersedes it. For example, if version @1.1.0@ is released on 1st January 2025, version @1.0.X@ will be supported until at least 1st January 2026.
+
+Releases will be always be deprecated within 24 months of being superseded.
+
+h3(#sunset). Sunset date
+
+The sunset date is the date on which a version of an SDK or API will no longer function, at the end of the deprecation period. Ably will gradually reject requests made using a sunset version to avoid large-scale disruption where possible, however this isn't guaranteed.
```

Comment:
```suggestion
The sunset date is the date on which a version of an SDK or API will no longer function, at the end of the deprecation period. Ably will usually sunset features by a process of rejecting just a (gradually increasing) proportion of requests to avoid large-scale disruption where possible; however, this won't always be the case, so you should assume that features or library versions will stop working on the sunset date.
```